src/coreclr/src/vm/eeconfig.cpp

```diff
@@ -560,23 +551,6 @@ fTrackDynamicMethodDebugInfo = CLRConfig::GetConfigValue(CLRConfig::UNSUPPORTED_
 
 #endif //STRESS_HEAP
 
-#ifdef HOST_64BIT
```

Comment:
we should get rid of these in clrconfigvalues.h too

Reply:
That would conflict with https://github.com/dotnet/runtime/pull/35259. I will do that once that is merged.